server/config.go

```diff
@@ -89,46 +93,53 @@ const (
 	configClusteringReplicaMaxIdleWait      = "clustering.replica.max.idle.wait"
 	configClusteringReplicaFetchTimeout     = "clustering.replica.fetch.timeout"
 	configClusteringMinInsyncReplicas       = "clustering.min.insync.replicas"
+
+	configActivityStreamEnabled              = "activity.stream.enabled"
+	configActivityStreamPublicationTimeout   = "activity.stream.publication.timeout"
+	configActivityStreamPublicationAckPolicy = "activity.stream.publication.ack.policy"
```

Comment:
Bit of a nitpick, but I propose `publish` instead of `publication`.

Reply:
Sure. Should be fixed in f0a1f13.